try:
    print(x)
except:
    print("Error")
finally:
    print("completed")
